plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/ResourceMojo.java

```diff
@@ -459,22 +458,23 @@ private void lateInit() {
         if (runtimeMode.equals(RuntimeMode.openshift)) {
             Properties properties = project.getProperties();
             if (!properties.contains(DOCKER_IMAGE_USER)) {
-                String namespace = clusterAccess.getNamespace();
+                String namespace = this.namespace!= null && this.namespace.length() > 0 ?
```

Comment:
Add space between `namespace` and operator `!=`

Reply:
I recommend using `!namespace.isEmpty()` rather than check size

Reply:
Done